### PR TITLE
Created ci pipeline to build and lint

### LINKED ISSUE #5: Set up CI
This can just be a basic build/test/link for PRs and such. It doesn't need to do deployment since that poses a security risk.

### PR DESCRIPTION
**Pull request by DipayanP007**
Closes #5 
Created a CI pipeline which:

* Builds the project
* performs liniting

### CODE REVIEW

File: .github/workflows/ci_build.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - uses: actions/checkout@v2
+    
+    - uses: actions-rs/toolchain@v1
+      with:
+          profile: minimal
+          toolchain: stable
+          override: true
+    - uses: actions-rs/cargo@v1
+      with:
+          command: build
```
Comment (multimeric) [inline .github/workflows/ci_build.yml:25]: `check` is preferred to `build` for CIs

**Review by multimeric — CHANGES_REQUESTED** [review]

**Comment by DipayanP007** [comment]
Hey @multimeric so should I replace `build` totally with `check` or should it have both `build` and `check`

**Comment by multimeric** [comment]
`check` instead of `build`.

**Comment by DipayanP007** [comment]
Hi @multimeric I have done the changes requested. kindly review it

**Review by multimeric — APPROVED** [review]

**Comment by multimeric** [comment]
Looks great, thanks!

**Pull request merged**